raiden/network/transport/matrix/client.py

```diff
@@ -30,7 +31,9 @@
 
 def node_address_from_userid(user_id: Optional[str]) -> Optional[AddressHex]:
     if user_id:
-        return to_checksum_address(user_id.split(":", 1)[0][1:])
+        address = user_id.split(":", 1)[0][1:]
+        if is_address(address):
```

Comment:
This is not necessary, `to_normailzed_address()` already performs an `is_address()` check itself.